xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/EnableAmendmentTest.java

```diff
@@ -0,0 +1,37 @@
+package org.xrpl.xrpl4j.model.transactions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.primitives.UnsignedInteger;
+import com.google.common.primitives.UnsignedLong;
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+
+import java.util.Optional;
+
+public class EnableAmendmentTest {
+
+  @Test
+  public void testJson() throws JsonProcessingException, JSONException {
```

Comment:
This test doesn't appear to be testing any JSON.  Let's rename the test to something clearer, like this:

```suggestion
  public void testBuilder() throws JsonProcessingException, JSONException {
```